libs/portals/my-pages/assets/src/screens/Overview/Overview.tsx

```
@@ -98,111 +85,58 @@ const VehiclesOverview = () => {
+            items={[
+              {
+                onClick: () => formSubmit(ownershipLinks.pdf),
+                title: formatMessage(messages.myCarsFilesPDF),
+              },
+              {
+                onClick: () =>  formSubmit(ownershipLinks.excel),
+                title: formatMessage(messages.myCarsFilesExcel),
+              },
+            ]}
+            title={formatMessage(messages.myCarsFiles)}
+          /> : undefined,
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)